src/api/app/assets/javascripts/webui2/project_monitor.js

```diff
@@ -6,18 +6,75 @@ function setAllLinks(event) {
   $(this).closest('.dropdown-menu').find('input').prop('checked', event.data.checked);
 }
 
-function setupProjectMonitor() { // jshint ignore:line
+function statusCell(meta, statusHash, tableInfo, projectName, packageName) {
+  var info = tableInfo[meta.col - 1];
+  var repository = info[0];
+  var architecture = info[1];
+  var status = statusHash[repository][architecture][packageName] || {};
+  var code = status.code;
+  if (code === undefined) { return null; }
+
+  var output = '<a ';
+  if (['succeeded', 'failed', 'building'].includes(code)) {
+    var url = '/package/live_build_log/' + projectName + '/' + packageName + '/' + repository + '/' + architecture;
+    output += 'href="' + url + '"';
+  }
+  else {
```

Comment:
More of a personal preference... I would prefer `} else {`. I think it's also how we write this elsewhere

Reply:
the linter does not complain.... I don't think style should be a topic we take care of. Although I like it more as you say as well...